commands/article.js

```diff
@@ -0,0 +1,28 @@
+const config = require('../config.json');
+const Parser = require('rss-parser');
+const parser = new Parser();
+const random = require('random');
```

Comment:
I don't think we need to add another dependency for this, can you implement this with `Math.random()` instead?